### ISSUE TITLE
macOS menu bar icon is huge

### ISSUE THREAD

**Issue by makitsune** [Hyperchroma/hyperchroma#4]
**Describe the bug**
The menu bar icon is really big on macOS. I've had this same problem too with my electron app and this has helped me get it right for Window, macOS and Linux:

-   https://stackoverflow.com/a/60940295
-   https://www.electronjs.org/docs/latest/api/native-image#high-resolution-image

I just use a 16x16 `tray-icon.png` which works fine but you could add bigger sizes with `tray-icon@2x.png`

It might also be important to note that macOS uses black or white icons depending on the current background and theme.

https://www.electronjs.org/docs/latest/api/native-image#template-image

I was playing around myself and seeing others' examples and this seems to be the correct way to do it:

```js
const tray = new Tray(
    process.platform == "darwin" ?
        "trayIconTemplate.png" :
        "trayIcon.png",
);
```

![Screen Shot 2021-11-27 at 20 18 44](https://user-images.githubusercontent.com/8362329/143726311-13c4b3f4-52c0-4a18-bbae-207c28fea218.png)

**Screenshots**
![Screen Shot 2021-11-27 at 19 14 53](https://user-images.githubusercontent.com/8362329/143725684-f9a41653-20e8-45b9-8b91-3245f89c34d2.png)

**Additional Information (please complete the following information):**
 - OS: Using macOS Monterey on an M1 MacBook

**Comment by makitsune** [comment]
Oh I didn't mention but when testing with `xxxTemplate.png` suffix, when changing my background it successfully changes to black or white as expected

**Comment by aeroheim** [comment]
Appreciate the detailed guidelines! Made the process a lot easier. 

Issue should be resolved in the 0.9.1-beta release. Please re-open if you still see the issue.
